package.json

```diff
@@ -1,17 +1,32 @@
 {
   "name": "wdio-common",
-  "version": "1.0.0",
+  "version": "2.0.0",
   "description": "Common helpers and components for WDIO projects",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test:chrome": "wdio config/chrome.conf.js"
   },
   "repository": {
     "type": "git",
     "url": "git+https://github.com/rsksmart/wdio-common.git"
   },
-  "author": "javicoin",
+  "author": "IOV Labs QA Team",
   "license": "ISC",
   "devDependencies": {
-    "chai": "^4.3.7"
+    "chai": "^4.3.7",
+
+
```

Comment:
Dependencies to be deleted from here and to be set just in the upper testing layer once everything is stable